platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml

```diff
@@ -79,15 +79,13 @@ network:
       # Git Repo details which will be used by GitOps/Flux.
       # Do not check-in git_access_token
       gitops:
-        git_ssh: "ssh://git@github.com/<username>/blockchain-automation-framework.git"         # Gitops ssh url for flux value files 
+        git_https: "https://github.com/<username>/blockchain-automation-framework.git"         # Gitops https url for flux value files 
```

Comment:
rename to git_url, and add a flag ssh: false. And create a separate story for ssh:true which will use ssh URL and private_key instead of password

Reply:
@sownak Done with the suggested changes. please review.